xcode/Subconscious/Shared/Components/Notebook/NotebookNavigationView.swift

```diff
@@ -41,12 +41,13 @@ struct NotebookNavigationView: View {
                 )
                 .ignoresSafeArea(.keyboard, edges: .bottom)
                 .confirmationDialog(
-                    "Are you sure?",
+                    "Are you sure you want to delete?",
```

Comment:
Can we fit "Are you sure you want to delete _this note_?" I tend to prefer confirmations to be specific about the result of the action.

Reply:
Good call.